chapter12/qli/ex2.pl

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+my $oldest_file;
+my $oldest_age=0;
+
+foreach my $file (@ARGV) {
+  if( -e $file && -M $file > $oldest_age ){
+    $oldest_file = $file;
+    $oldest_age = -M $file;
+  }else{
```

Comment:
That's very good point. When I wrote that line, I probably think I may have more code after that if statement.